libvmi/os/windows/peparse.c

```diff
@@ -516,7 +516,25 @@ peparse_get_export_table(
     nbytes = vmi_read_va(vmi, export_header_va, pid, et, sizeof(*et));
     if (nbytes != sizeof(struct export_table)) {
         dbprint(VMI_DEBUG_PEPARSE, "--PEParse: failed to map export header\n");
-        return VMI_FAILURE;
+
+        /*
+         * Sometimes Windows maps the export table on page-boundaries,
+         * such that the first export_flags field (which is reserved) is.
+         * not actually accessible (the page is not mapped). See Issue #260.
+         */
+        if (export_header_va >> 12 != (export_header_va+4) >> 12) {
```

Comment:
@awsaba: sure. The check you propose is a bit more restrictive but it does match what we currently see so I'll make the adjustment.